sources/packages/api/src/database/entities/sfas-part-time-application.model.ts

```diff
@@ -0,0 +1,68 @@
+import { Column, Entity, PrimaryColumn } from "typeorm";
+import { BaseModel } from ".";
+import { TableNames } from "../constant";
+import { dateOnlyTransformer } from "../transformers/date-only.transformer";
+
+/**
+ * Data related to a Student Application on SFAS Part Time.
```

Comment:
Can we update this comment to "Data related to a Part Time Student Application on SFAS that came from SAIL"?
This is confusing because the data is coming from SFAS integration but PART TIME data leaves on SAIL, so it would be better to have it expressed in the comments.